resources/views/landing.blade.php

```diff
@@ -1,42 +1,27 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="de">
```

Comment:
Maybe output the current locale instead?

Reply:
This is merely a placeholder. It's outputting the correct locale.